core/tests/protractor_utils/forms.js

```diff
@@ -343,9 +357,10 @@ var AutocompleteMultiDropdownEditor = function(elem) {
       }
```

Comment:
Can we use `action.click()` above?

Reply:
updated!

Reply:
> @tigerdw Hi, do you need help with this PR?

I am currently reviewing and updating with the comments!